test/simple_mercator/polygon_test.rb

```diff
@@ -15,6 +15,14 @@ def setup
     @factory = RGeo::Geographic.simple_mercator_factory
   end
 
+  def test_is_simple_validation_behavior
+    # issue 218
```

Comment:
I always find it easier to show an url in that case:
```suggestion
    # See https://github.com/rgeo/rgeo/issues/218
```